analysis/summarise_consultation_dataset.R

```diff
@@ -25,23 +25,50 @@ consultation_datasets <- consultation_dataset_paths %>%
 
 # Count sum and has (yes/no) of consultations grouped by age group
 # We could add more variables to group by here, e.g., ethnicity or sex
+# df_summary <- consultation_datasets %>%
+#   dplyr::group_by(start_date, end_date, age_greater_equal_65) %>%
+#   dplyr::summarise(
+#     n_sum_f2f = sum(count_f2f_consultation, na.rm = TRUE),
+#     n_sum_virtual = sum(count_virtual_consultation, na.rm = TRUE),
+#     n_sum_appointment = sum(count_appointments, na.rm = TRUE),
+#     n_has_f2f = sum(has_f2f_consultation, na.rm = TRUE),
+#     n_has_virtual = sum(has_virtual_consultation, na.rm = TRUE),
+#     n_has_appointment = sum(has_appointments, na.rm = TRUE),
+#   ) %>%
+#   pivot_longer(
+#     cols = c(
+#       n_sum_f2f, n_sum_virtual, n_sum_appointment,
+#       n_has_f2f, n_has_virtual, n_has_appointment
+#     ),
+#     names_to = c("summary_type", "consultation_type"),
+#     values_to = "value", names_sep = "_(?=f2f|virtual|appointment)"
+#   )
+
+#Count sum and has (yes/no) of consultations grouped by age group
+#We could add more variables to group by here, e.g., ethnicity or sex
```

Comment:
@AbodunrinAminu-Manchester can we delete this or is there a reasons you kept this commented out?

Reply:
Yes we can delete it. I commented it out just to remind myself what we had previously before I modified the codes.